.github/workflows/ci-cd.yml

```diff
@@ -138,3 +138,48 @@ jobs:
 
   #     - name: Test
   #       run: yarn test:ember-compatibility ${{ matrix.scenario }} --- yarn lint:types --project tsconfig.ts3.json
+
+
+  deploy-documentation:
+    name: Deploy documentation
+    runs-on: ubuntu-latest
+    needs: [lint, test-addon, test-floating-dependencies, try-scenarios]
```

Comment:
To be safe, I configured the `deploy-documentation` job to occur only after all 4 jobs pass. Theoretically, passing `lint` and `test-addon` may be deemed sufficient for deploying a documentation site (thereby, triggering the `deploy-documentation` job sooner).